src/components/time-entries/TimeEntries.tsx

```diff
@@ -11,6 +12,16 @@ export const TimeEntries = ({ isModalActive, handleModal }: Types.AtBuildProps)
   const { timeEntries, setTimeEntries } = useContext(EntriesContext);
   const [sortOrder, setSortOrder] = useState("ascending");
   const [sortedTimeEntries, setSortedTimeEntries] = useState(timeEntries);
+  const [selectedClient, setSelectedClient] = useState("");
+
+  const filterClients = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectedClient(event.target.value);
+  };
+
+  const timeClients = timeEntries.map((entry) => entry.client);
+  const uniqueClients = timeClients.filter((client, index) => {
```

Comment:
Another nice way to get unique values from an array is by using `Set`: `const uniqueClients = Array.from(new Set(timeClients))`.